notebooks/00_quick_start/run_notebook_on_aml.ipynb

```diff
@@ -0,0 +1,2380 @@
+{
```

Comment:
<p>Can we split this cell into 2, one for setting the experiment configuration and another for executing the notebook</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/681/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Done